apps/src/code-studio/components/Libraries/LibraryClientApi.js

```diff
@@ -61,8 +63,53 @@ export default class LibraryClientApi {
     });
   }
 
-  getClassLibraries(onSuccess, onError) {
-    // TODO: find all libraries published by other students in a classroom
-    return undefined;
+  async getClassLibraries(onSuccess, onError) {
```

Comment:
No tests for this because it is a short-term fix with an high-priority replacement: https://codedotorg.atlassian.net/browse/STAR-746